Systems/instruments.xml

```diff
@@ -359,5 +359,24 @@
             <property>/sim/model/c172p/lighting/trans</property>
         </output>
     </filter>
+    
+    <filter>
+        <name>Clock: UTF to local time conversion</name>
+        <type>gain</type>
+        <input>
+            <expression>
```

Comment:
Instead of using `<expression>` you could try:

```xml
<input>
    <property>sim/time/utc/day-seconds</property>
    <offset>
        <property>sim/time/local-offset</property>
    </offset>
</input>
```